### DISCUSSION TITLE
How pgsc_calc handles missing genotype data

### DISCUSSION THREAD

**Discussion by kentaro0209** [PGScatalog/pgsc_calc#307] [Q&A] [answered]
Hello,
I have a question about how pgsc_calc handles missing genotype data. Even after imputation, there may be some missing SNPs in a cohort. I'm wondering how pgsc_calc handles these missing SNPs. Pgsc_calc ignores them, or uses average/mean imputation rate or something? 
I'd appreciate it if you could some comments on this question.

Thanks,
Kentaro Takahashi

**Comment by smlmbrt** [comment, marked as answer]
Extract from https://github.com/PGScatalog/pgsc_calc/discussions/305#discussioncomment-9596705:
>The current pipeline (v2.0.0) in --run_ancestry mode will use the allele frequency in the reference panel to impute the missing allele dosage during PGS Calculation, in the normal mode it will use the allele frequency in your samples to impute the dosage (and also calculate the AVG PGS for non-missing sites).

**Reply by Fiwx** [reply]
> run_ancestry mode will use the allele frequency in the reference panel

Is it correct that this is not ancestry-specific? That is, e.g. for 1000G, it just uses all samples in the file.

**Reply by smlmbrt** [reply]
Yes, you're correct @Fiwx - it uses the AF across the entire reference panel. It would increase the number of plinks calls by N populations in the target samples if we were to use only those AFs as input.
